can't install by npm 5.4.0 with error below

```
> chromedriver@2.32.2 install /Users/kevinaskin/WebstormProjects/kevinaskin/live/public/node_modules/chromedriver
> node install.js

Downloading https://npm.taobao.org/mirrors/chromedriver/2.32/chromedriver_mac64.zip
Saving to /var/folders/6c/mz_lxxxj0kxbqxkgd910tnsr0000gn/T/chromedriver/chromedriver_mac64.zip
ChromeDriver installation failed Error with https: request: Error: Protocol "https:" not supported. Expected "http:"
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! chromedriver@2.32.2 install: `node install.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the chromedriver@2.32.2 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
```


and `.npmrc` includes: 
```
chromedriver_cdnurl = "https://npm.taobao.org/mirrors/chromedriver"
strict-ssl = false
```

PS:
* when I use `cnpm install`, it is okay to install successfully
* I check this path `/var/folders/6c/mz_lxxxj0kxbqxkgd910tnsr0000gn/T/chromedriver/chromedriver_mac64.zip`,
this file is exists already

--- a/install.js
+++ b/install.js
@@ -114,9 +114,9 @@ function findSuitableTempDirectory() {
 
 
 function getRequestOptions(downloadPath) {
-  var options = {uri: downloadPath};
-  options.protocol = options.uri.substring(0, options.uri.indexOf('//'));
-  var proxyUrl = options.protocol === 'https:'
+  var options = {uri: downloadPath, method: 'GET'};
+  var protocol = options.uri.substring(0, options.uri.indexOf('//'));
+  var proxyUrl = protocol === 'https:'
     ? process.env.npm_config_https_proxy
     : (process.env.npm_config_proxy || process.env.npm_config_http_proxy);
   if (proxyUrl) {
@@ -167,9 +167,9 @@ function getRequestOptions(downloadPath) {
 
 function getLatestVersion(requestOptions) {
   var deferred = kew.defer();
-  request.get(requestOptions, function (err, response, data) {
+  request(requestOptions, function (err, response, data) {
     if (err) {
-      deferred.reject('Error with ' + requestOptions.protocol + ' request: ' + err);
+      deferred.reject('Error with http(s) request: ' + err);
     } else {
       chromedriver_version = data.trim();
       deferred.resolve(true);
@@ -185,10 +185,10 @@ function requestBinary(requestOptions, filePath) {
   var notifiedCount = 0;
   var outFile = fs.openSync(filePath, 'w');
 
-  var client = request.get(requestOptions);
+  var client = request(requestOptions);
 
   client.on('error', function (err) {
-    deferred.reject('Error with ' + requestOptions.protocol + ' request: ' + err);
+    deferred.reject('Error with http(s) request: ' + err);
   });
 
   client.on('data', function (data) {

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "chromedriver",
-  "version": "2.32.2",
+  "version": "2.32.3",
   "keywords": [
     "chromedriver",
     "selenium"

